exporter/sentryexporter/sentry_exporter.go

```diff
@@ -35,25 +35,11 @@ const (
 )
 
 // canonicalCodes maps OpenTelemetry span codes to Sentry's span status.
-// See numeric codes in https://godoc.org/github.com/open-telemetry/opentelemetry-proto/gen/go/trace/v1#Status_StatusCode.
+// See numeric codes in https://github.com/open-telemetry/opentelemetry-proto/blob/6cf77b2f544f6bc7fe1e4b4a8a52e5a42cb50ead/opentelemetry/proto/trace/v1/trace.proto#L303
 var canonicalCodes = [...]string{
+	"unset",
 	"ok",
-	"cancelled",
-	sentryStatusUnknown,
-	"invalid_argument",
-	"deadline_exceeded",
-	"not_found",
-	"already_exists",
-	"permission_denied",
-	"resource_exhausted",
-	"failed_precondition",
-	"aborted",
-	"out_of_range",
-	"unimplemented",
-	"internal",
-	"unavailable",
-	"data_loss",
-	"unauthenticated",
+	"error",
```

Comment:
Sentry relies on exact values for these status codes as per: https://github.com/getsentry/relay/blob/1a00926a6e8936173d4281777c27814d509a9fe5/relay-common/src/constants.rs#L254

To get around this, we should probably map `unset` and `error` to `unknown` for now. If/when Sentry accommodates for the otel spec change for span status, I can come back and update the exporter.

cc @rhcarvalho so Sentry folks are aware of this change.

Reply:
Done.